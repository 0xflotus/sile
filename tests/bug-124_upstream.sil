\begin[papersize=a5]{document}
\script[src=packages/bidi]
\set[parameter=document.parindent,value=0]
\set[parameter=current.parindent,value=0]
\font[family=Scheherazade,size=40pt,language=ar,script=Arab]
\thisframeRTL
\font[family=Noto Naskh Arabic]{ألمّ} — \font[family=Scheherazade]{ألمّ

— \glue[width=20pt 0 0]إِنْ

— السُّــرورُ}
\end{document}
