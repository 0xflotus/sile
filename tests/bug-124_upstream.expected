<tests/bug-124_upstream.sil>
Set paper size 	419.5275636	595.275597
Begin page
Mx 	359.43009
My 	66.75597
Set font 	Noto Naskh Arabic;40;200;regular;normal;;RTL
T		(ألمّ)
Mx 	318.29717
Set font 	Scheherazade;40;200;regular;normal;;RTL
T	120	(—)
Mx 	279.00019
T		(ألمّ)
Mx 	369.54728
My 	113.95128
T	120	(—)
Mx 	315.50548
T		(إِنْ)
Mx 	369.54728
My 	163.33409
T	120	(—)
Mx 	286.88698
T		(السُّــرورُ)
[1] Mx 	207.41759
My 	553.73265
Set font 	Gentium;10;200;regular;normal;;LTR
T	17	(1)
End page
Finish

